arboretum/permissions/README.md

```diff
@@ -97,10 +97,73 @@ day.
 
 ## Checks
 
-Checks coming soon...
+### Organization Integrity (Repository Collaborators)
+
+* Class: [OrgCollaboratorsCheck][org-collaborators-check]
+* Purpose: Ensure that `direct` collaborators do not exist in the organization repositories.
+* Behavior: Collaborators are checked for every repository.
+A failure is generated when direct collaborators are found in a repository. This check can be optionally
+configured to accept exceptions, a warning instead of a failure is generated for those exceptions when
+direct collaborators matching the exceptions are found.
+* Evidence depended upon:
+    * Recent `direct` collaborators found in organization repositories.
```

Comment:
What do you mean by "Recent"?

Reply:
Agreed that the word `Recent` can be removed. I will fix.